lib/msal-core/src/Utils.ts

```diff
@@ -705,6 +705,7 @@ export class Utils {
   static setResponseIdToken(originalResponse: AuthResponse, idToken: IdToken) : AuthResponse {
     var response = { ...originalResponse };
     response.idToken = idToken;
+    response.claims = idToken.decodedIdToken;
```

Comment:
Same comment as above.

Reply:
Done. As agreed, changed idToken in response to "String" and added "claims" in 'reponse' and 'account' objects.